exo-go/src/app_deployer/index.go

```diff
@@ -12,27 +14,51 @@ import (
 type AppDeployer struct {
 	AppConfig      types.AppConfig
 	ServiceConfigs map[string]types.ServiceConfig
-	Logger         *logger.Logger
 	AppDir         string
 	HomeDir        string
+	Logger         *logger.Logger
 }
 
 // NewAppDeployer is the constructor for the appDeployer class
-func NewAppDeployer(appConfig types.AppConfig, serviceConfigs map[string]types.ServiceConfig, appDir, homeDir string) *AppDeployer {
+func NewAppDeployer(appConfig types.AppConfig, serviceConfigs map[string]types.ServiceConfig, appDir, homeDir string, logger *logger.Logger) *AppDeployer {
 	return &AppDeployer{
 		AppConfig:      appConfig,
 		ServiceConfigs: serviceConfigs,
 		AppDir:         appDir,
 		HomeDir:        homeDir,
+		Logger:         logger,
 	}
```

Comment:
There isn't much point for having a constructor unless you are actually performing some initialization logic. If you are just assigning public properties then you don't need one as that can just be done directly